packages/koa-shopify-auth/src/auth/redirection-page.ts

```diff
@@ -12,7 +12,7 @@ export default function redirectionScript({origin, redirectTo, apiKey}) {
           var Redirect = AppBridge.actions.Redirect;
           var app = createApp({
             apiKey: '${apiKey}',
-            shopOrigin: '${origin}',
+            shopOrigin: '${encodeURI(origin)}',
```

Comment:
```suggestion
            shopOrigin: "${encodeURI(origin)}",
```
Would suggest wrapping in double quotes here, since `encodeURI()` doesn't encode single quotes. So getting an input like `shop='syntaxerror` could still cause some brokenness.

Reply:
Unless I'm missing something here I think wrapping in anything other than backticks will break this templating